lib/services/access_request.go

```diff
@@ -36,20 +36,25 @@ import (
 	apidefaults "github.com/gravitational/teleport/api/defaults"
 	"github.com/gravitational/teleport/api/types"
 	apiutils "github.com/gravitational/teleport/api/utils"
-	"github.com/gravitational/teleport/lib/defaults"
 	"github.com/gravitational/teleport/lib/tlsca"
 	"github.com/gravitational/teleport/lib/utils"
 	"github.com/gravitational/teleport/lib/utils/parse"
 )
 
-const maxAccessRequestReasonSize = 4096
+const (
+	maxAccessRequestReasonSize = 4096
 
-// A day is sometimes 23 hours, sometimes 25 hours, usually 24 hours.
-const day = 24 * time.Hour
+	// A day is sometimes 23 hours, sometimes 25 hours, usually 24 hours.
+	day = 24 * time.Hour
 
-// maxAccessDuration is the maximum duration that an access request can be
-// granted for.
-const maxAccessDuration = 7 * day
+	// maxAccessDuration is the maximum duration that an access request can be
+	// granted for.
+	maxAccessDuration = 7 * day
+
+	// requestTTL is the the TTL for an access request, i.e. the amount of time that
+	// the access request can be reviewed. Defaults to 1 week.
+	requestTTL = 7 * day
```

Comment:
Is there a way to override this TTL, or is it always 7 days no matter what?

If the latter, then it feels disingenuous to say this "Defaults to 1 week" - default typically means "the value that is used if not otherwise specified."

Reply:
There is a way to override it with the `maxDuration` flag. I'm not too familiar with this flag and how it works, however.

Reply:
IIUC we're here setting the object expiry (`SetExpiry`) to 7 days. Below, we use `maxDuration` to potentially modify `SetSessionTTL`, `SetAccessExpiry`, and `SetMaxDuration`, but not `SetExpiry`. Is there something I'm missing? If not, I think we should clarify the relationship between these settings and this `maxDuration` before making these changes.